Modules/Loadable/TortuosityModule/Logic/vtkSlicerTortuosityLogic.cxx

```diff
@@ -262,13 +266,21 @@ ::RunMetrics(vtkMRMLSpatialObjectsNode* node, int flag)
       continue;
       }
 
+    VesselTubeType::Pointer smoothedTube = VesselTubeType::New();
+    VesselTubeType::Pointer subsampledTube = VesselTubeType::New();
+
+    // Smooth the vessel
+    smoothedTube = tube::SmoothTube<VesselTubeType>(currTube, smoothingScale, (tube::SmoothTubeFunctionEnum)smoothingMethod );
+    // Subsample the vessel
+    subsampledTube = tube::SubsampleTube<VesselTubeType>(smoothedTube, subsampling);
+
     // Update filter
     FilterType::Pointer filter = FilterType::New();
     filter->SetMeasureFlag(flag);
-    filter->SetInput(currTube);
+    filter->SetInput(subsampledTube);
     filter->Update();
 
-    if (filter->GetOutput()->GetId() != currTube->GetId())
+    if (filter->GetOutput()->GetId() != subsampledTube->GetId())
```

Comment:
Formatting of the if statement should be
if( filter->GetOutput().... )
- No space between if and right paren
- Space after right paren
- Space before left paren